zip-1014.rst

```diff
@@ -338,6 +288,14 @@ All substantial software whose development was funded by the Dev Fund SHOULD
 be released under an Open Source license (as defined by the Open Source
 Initiative [#osd]_), preferably the MIT license.
 
+Dev Fund ZEC that has been received and has not yet been used or disbursed
+SHALL be kept by the corresponding party (as ZEC, or sold and invested) for
+later use under the terms of the corresponding slice. The parties are each
```

Comment:
Should this explicitly say "ECC and ZF" rather than "The parties"? Will the Major Grant Committee (if it exists) be a party?

Reply:
The MG committee is not directly keeping funds, it is instructing ZF how to disburse the MG funds.
And I assume we don't intend this to apply to MG _recipients_.
So yes, "ECC and ZF" is better.